Project.toml

```
@@ -28,7 +28,7 @@ ClimaComms = "0.3, 0.4"
 ClimaCore = "0.10"
 ClimaCoreTempestRemap = "0.3.5, 0.3.6, 0.3.7"
 DocStringExtensions = "0.8, 0.9"
-Insolation = "0.3"
+Insolation = "0.3, 0.4, 0.5"
```

```suggestion
Insolation = "0.5"
```
The current code requires passing an argument to `OrbitalData`, so the state of the code is incompatible with previous versions of Insolation.jl

---

And let's apply this same compat to all of the relevant environments